hangman.c

```diff
@@ -0,0 +1,312 @@
+//Olivier Gabison - Hangman Game
+//NOTE: I don't understand the error here as well?
```

Comment:
Be more careful with how you code and test your code more often! You made a lot of careless mistakes, which you got stuck on and ultimately prevented you from successfully completing the program.